docs/tutorial/graphical.rst

```diff
@@ -338,5 +338,66 @@ This adds the input line above the Queue and Abort buttons.
 
 A completer is implemented allowing to quickly select an existing folder, and a button on the right side of the input widget opens a browse dialog.
 
+Using the estimator widget
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In order to provide estimates of the measurement procedure, an EstimatorWidget is provided that allows the user to define and calculate estimates.
+The widget is automatically activated when the :code:`get_estimates` method is added in the :code:`Procedure`.
+
+The quickest and most simple implementation of the :code:`get_estimates` function simple returns a the estimated duration of the measurement in seconds (as an :code:`int` or a :code:`float`).
+As an example, in the example provided in the ManagedWindow section, the :code:`Procedure` is changed to:
+.. code-block:: python
+
+   class RandomProcedure(Procedure):
+
+       # ...
+
+       def get_estimates(self, sequence_length=None, sequence=None):
+
+           return self.iterations * self.delay
+
+This will add the estimator widget at the dock on the left.
+The duration and finishing-time of a single measurement is always displayed in this case.
+Depending on whether the SequencerWidget is also used, the length, duration and finishing-time of the full sequence is also shown.
+
+For maximum flexibility (e.g. for showing multiple and other types of estimates, such as the duration, filesize, finishing-time, etc.) it is also possible that the :code:`get_esimates` returns a list of tuples.
+Each of these tuple consists of two strings: the first is the name (label) of the estimate, the second is the estimate itself.
+
+As an example, in the example provided in the ManagedWindow section, the :code:`Procedure` is changed to:
+
```

Comment:
Is it possible to change ManagedWindow such that you have a cross reference link to the mentioned section ?

Reply:
Yes it is; fixed it.